website/cue/reference/components/sources/kafka.cue

```diff
@@ -132,7 +132,7 @@ components: sources: kafka: {
 			required:    false
 			warnings: []
 			type: string: {
-				default: null
+				default: "headers"
```

Comment:
I think this was wrong before (I didn't change the behavior for the source)

Reply:
I think you're right.